src/resources/BreadCrumbResource.ts

```diff
@@ -10,22 +9,18 @@ const store = useStore()
 
 /** Returns legal name. */
 function getLegalName (): string {
-  const getFilingType: FilingTypes = store.getFilingType
-  const getBusinessLegalName: string = store.getBusinessLegalName
-  const getNameRequestApprovedName: string = store.getNameRequestApprovedName
```

Comment:
`getFilingType` was only used once so I moved it there.

`getBusinessLegalName` was only needed in some cases so I moved it there.

`getNameRequestApprovedName` was only needed in some cases so I moved it there.

^^ Also it looked nice to assign variables ahead of time, in all cases some variables weren't used, so I simplified this.